spec/formatting.md

```diff
@@ -489,6 +489,12 @@ Earlier _selectors_ in the _matcher_'s list of _selectors_ have a higher priorit
 When all of the _selectors_ have been processed,
 the earliest-sorted _variant_ in the remaining list of _variants_ is selected.
 
+> [!NOTE]
+> A _selector_ _expression_ is not a _declaration_,
+> and its _annotation_ has no effect on the value of its _operand_ during formatting.
```

Comment:
I think the term "expression" isn't needed here. We use just "selector" all through this section otherwise.

I think the term "value" has also been a source of woe for us of late and I'd rather avoid it if we can. Perhaps:

```suggestion
> A _selector_ is not a _declaration_.
> Even when the same _function_ can be used for both formatting and selection
> of a given _operand_
> the _annotation_ that appears in a _selector_ has no effect on subsequent
> _selectors_ nor on the formatting used in _placeholders_.
```